addon/templates/components/polaris-card.hbs

```diff
@@ -3,7 +3,11 @@
     <div class="Polaris-Card__Header">
       {{#polaris-stack alignment="baseline" as |stack|}}
         {{#stack.item fill=true}}
-          {{polaris-heading text=title}}
+          {{#if (is-component-definition title)}}
```

Comment:
Hmm... this helper is a nice idea, but it will ignore the changes I made to `render-content` in #147 (handling `content` being a hash of `componentName` and `props`). We probably want to be able to support both the component definition and `componentName` + `props` hash way of doing things here, since `render-content` will support it...

Reply:
not really an idea...we need to not wrap title in a heading if it's passed as a component....but will check if we can support both